scalereg/reg6/views.py

```diff
@@ -23,6 +23,8 @@
 
 PGP_KEY_QUESTION_INDEX_OFFSET = 3
 
+KIOSK_AGENT = 'CHANGE_ME:'
```

Comment:
Should this come from settings.py instead?

Reply:
Done.